<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfirmPropMeasureLineMoveForm</class>
 <widget class="QWidget" name="ConfirmPropMeasureLineMoveForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QFrame" name="frame_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>481</width>
     <height>101</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="labelMeasureLineMoveDistanceLengthUnit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>170</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelLength">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>計測値</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelUnitName_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>mm</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_40">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>間隔測長線</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_93">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>長さ変動　下限しきい値（最小０）</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelUnitName_9">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMeasureLineMoveDistanceThresholdP">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMeasureLineMoveDistanceThresholdPMM">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="maximum">
    <double>9999999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMeasureLineMoveDistanceThresholdM">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximum">
    <double>999999.989999999990687</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_94">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>長さ変動　上限しきい値（最小０）</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMeasureLineMoveDistanceThresholdMMM">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>91</width>
     <height>28</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="maximum">
    <double>9999999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxMeasureLineMoveDistanceModeToSetInInitial">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Update position and value in Initialization</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_96">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>381</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>初期化時にOK範囲を設定するときのOK幅(mm)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMeasureLineMoveDistanceOutputType">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>計測値の出力</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>しきい値中央値との差分出力</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelUnitName_8">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Dot</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_97">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>出力タイプ</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMeasureLineMoveDistanceOKRangeInInitial">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>160</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_33">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>491</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>計測ポイント線</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="EditMeasureLineMoveLineEdgeWidth1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_92">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>探索ドット</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelUnitName_5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>330</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Dot</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="EditMeasureLineMoveLineSearchDot1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMeasureLineMoveLineSearchDot1MM">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="maximum">
    <double>9999999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_95">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>輪郭幅</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelUnitName_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMeasureLineMoveLineUsedLayer1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_100">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>420</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Used layer</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="EditMeasureLineMoveLineEdgeWidth2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>390</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="EditMeasureLineMoveLineSearchDot2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>360</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBoxMeasureLineMoveLineSearchDot2MM">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>360</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>170</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>240</red>
         <green>240</green>
         <blue>240</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="maximum">
    <double>9999999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxMeasureLineMoveLineUsedLayer2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>420</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelUnitName_6">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>330</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Dot</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelUnitName_10">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>330</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>mm</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_98">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>300</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>計測線１</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_99">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>300</y>
     <width>151</width>
     <height>28</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>計測線２</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
